connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockInventoryTransactionTranslator.java

```diff
@@ -109,6 +117,46 @@ public void translate(InventoryTransactionPacket packet, GeyserSession session)
                             break;
                         }
 
+                        Vector3i blockPos = BlockUtils.getBlockPosition(packet.getBlockPosition(), packet.getBlockFace());
+                        /*
+                        Checks to ensure that the range will be accepted by the server.
+                        "Not in range" doesn't refer to how far a vanilla client goes (that's a whole other mess),
+                        but how much a server will accept from the client maximum
+                         */
+                        // CraftBukkit+ check - see https://github.com/PaperMC/Paper/blob/458db6206daae76327a64f4e2a17b67a7e38b426/Spigot-Server-Patches/0532-Move-range-check-for-block-placing-up.patch
```

Comment:
Would it be better referencing the master version of this file in case it gets updated/changed?
```suggestion
                        // CraftBukkit+ check - see https://github.com/PaperMC/Paper/blob/master/Spigot-Server-Patches/0532-Move-range-check-for-block-placing-up.patch
```

Reply:
No, because patch numbers get moved all the time - in a week to a month, that name will be outdated.